### ISSUE TITLE
Fontconfig unknown element/invalid attribute errors

### ISSUE THREAD

**Issue by ThatsNoMoon** [Automattic/node-canvas#1309]
## Issue
When drawing text, fontconfig throws out 100+ lines of errors & warnings about invalid attributes and unknown elements, ending with `Fontconfig error: Cannot load default config file`. Full log can be found [here](https://gist.github.com/ThatsNoMoon/4f88467fc9cd87579154e2a703aa618a).

Drawing text takes about 30 seconds to complete, and when it does it's always in my system's default monospace font, even if I attempt to change the font to something else.

Based on the answers given to people with similar problems [here](https://unix.stackexchange.com/questions/440320/fontconfig-error-when-running-octave) and [here](https://ask.fedoraproject.org/en/question/124425/fontconfig-error-invalid-attribute/), I'm guessing node-canvas is using an outdated version of fontconfig.

## Steps to Reproduce
```js
var Canvas = require('canvas');
var canvas = Canvas.createCanvas(200, 200);
var ctx = canvas.getContext('2d');
ctx.fillText("hi", 100, 100);
```
## Environment
* node-canvas version: 2.1.0
* Environment: Node 11.2.0 on Arch Linux
* Fontconfig: 2.13.1

**Comment by chearon** [comment]
We're distributing FontConfig [2.12.4](https://github.com/node-gfx/node-canvas-prebuilt/blob/master/docker/Dockerfile#L19). Are you able to draw text at all? Hopefully the errors are mostly harmless. It's a little disappointing that FontConfig config files aren't backwards compatible, at least as far as logging.

**Comment by ThatsNoMoon** [comment]
I can technically draw text, but it takes about 30 seconds and it's always in the default monospace font. Even after adding `ctx.font = "40px sans-serif"` to the above example, the resulting image still has a serif, monospace font, somewhat like my system's default bitmap font.

**Comment by gugu** [comment]
I had the same issue. Looks like prebuilt version is broken in your case. Try to run `npm install --build-from-source canvas`

**Comment by chearon** [comment]
That's the best solution right now.

The only idea I have for the long-term is to update the existing FontConfig binaries to the latest version. Newer FontConfig is probably (🤞) compatible with old conf files, but we know older FontConfig is not compatible with newer conf files... I'll start updating the Dockerfile.

**Comment by chearon** [comment]
@ThatsNoMoon I updated FontConfig to latest in the just-released 2.2.0. Mind trying it out again?

**Comment by ThatsNoMoon** [comment]
Yep, that seems to have resolved it for me. Thanks!
